src/test/java/ibm/jceplus/junit/openjceplusfips/TestAll.java

```diff
@@ -40,16 +36,6 @@
         TestSHA3_224.class, TestSHA3_256.class, TestSHA3_384.class, TestSHA3_512.class,
         TestRSAKeyInterop.class, TestRSAKeyInteropBC.class, TestRSAPSS2.class,
         TestFIPSVerifyOnlyTest.class, TestRSASignatureWithSpecificSize.class})
-
+@Suite
```

Comment:
Can we add an empty line here?

Reply:
New line added in commit bd27b562b46b98c872982261f4a7cb8bb2145955